src/test/java/uk/ac/cam/cl/dtg/util/mappers/MapperTestUtils.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.cam.cl.dtg.util.mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class MapperTestUtils {
+  private static final ObjectMapper jsonMapper = new ObjectMapper();
+
+  public static void assertDeepEquals(Object expected, Object actual) throws JsonProcessingException {
```

Comment:
Rather than throwing `JsonProcessingException`, capture that exception in this method and call `fail(String, Throwable)`.

This will also mean that you should remove `throws JsonProcessingException` from all the tests where this assertion is used.